jekyll/_cci2/add-ssh-key.adoc

```diff
@@ -32,8 +32,10 @@ See the following VCS-specific docs for additional details on creating SSH keys:
 CircleCI cannot decrypt SSH keys, therefore every new key must have an empty passphrase. The below examples are for macOS.
 
 . In a terminal, generate the key with `ssh-keygen -t ed25519 -C "your_email@example.com"`. See link:https://www.ssh.com/ssh/keygen/[Secure Shell documentation] for additional details.
-. In the CircleCI application, go to your project's settings by clicking the *Project Settings* button (top-right on the *Pipelines* page of the project).
-. On the *Project Settings* page, select *SSH Keys*.
+. In the link:https://app.circleci.com[CircleCI web app], select your org from the org cards on your user homepage.
```

Comment:
Wherever you have these three steps you can use the partial: `partials/app-navigation/steps-to-project-settings.adoc`

Reply:
Sorry I think I was unclear about this in the ticket